06_gpu_and_ml/llm-serving/trtllm_llama.py

```diff
@@ -567,7 +567,7 @@ def main():
 
 class GenerateRequest(pydantic.BaseModel):
     prompts: list[str]
-    settings: Optional[dict]
+    settings: Optional[dict] = None
```

Comment:
Added `= None` as otherwise this wasn't really optional

Reply:
In Python typing, `Optional` means "can be `None`" and is distinct from "optional argument", i.e. argument with a default.

Reply:
True, I guess we could change the `curl` example doc below instead to include a `settings = None` param if you prefer. With fastapi v2 either this has to be `None` or we need to pass it explicitly in the request, so one of the two places needs an update